specification/compute/resource-manager/Microsoft.Compute/RecommenderRP/tspconfig.yaml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  "service-dir":
+    default: "sdk/widget"
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    use-read-only-status-schema: true
+    omit-unreachable-types: true
+    emitter-output-dir: "{project-root}"
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{version-status}/{version}/RecommenderRP.json"
+    examples-dir: "{project-root}/examples"
+  "@azure-tools/typespec-csharp":
+    flavor: azure
+    package-dir: "Azure.ResourceManager.Widget"
+    clear-output-folder: true
+    model-namespace: true
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-python":
+    package-dir: "azure-mgmt-widget"
+    namespace: "azure.mgmt.widget"
+    generate-test: true
+    generate-sample: true
+    flavor: "azure"
+  "@azure-tools/typespec-java":
+    package-dir: "azure-resourcemanager-widget"
+    namespace: "com.azure.resourcemanager.widget"
+    service-name: "Widget" # human-readable service name, whitespace allowed
+    flavor: azure
+  "@azure-tools/typespec-ts":
+    package-dir: "arm-widget"
```

Comment:
We are not going to call the service name widget, please come up with a name that makes sense.